File: src/debug/java/burrows/apps/example/gif/DebugApp.java
package burrows.apps.example.gif;

import android.os.StrictMode;
import android.os.StrictMode.ThreadPolicy;
import android.os.StrictMode.VmPolicy;

/**
 * @author <a href="mailto:jaredsburrows@gmail.com">Jared Burrows</a>
 */
public class DebugApp extends App {
  @Override public void onCreate() {
    StrictMode.setThreadPolicy(new ThreadPolicy.Builder()
      .detectAll()
      .penaltyLog()
      .build());
    StrictMode.setVmPolicy(new VmPolicy.Builder()
      .detectAll()
      .penaltyLog()
      .build());

    super.onCreate();
  }
}

File: src/debug/java/burrows/apps/example/gif/DebugApp.kt
package burrows.apps.example.gif

import android.os.StrictMode
import android.os.StrictMode.ThreadPolicy
import android.os.StrictMode.VmPolicy

class DebugApp : App() {
  override fun onCreate() {
    StrictMode.setThreadPolicy(ThreadPolicy.Builder()
      .detectAll()
      .penaltyLog()
      .build())
    StrictMode.setVmPolicy(VmPolicy.Builder()
      .detectAll()
      .penaltyLog()
      .build())

    super.onCreate()
  }
}
